iam/snippets/src/main/java/ListServiceAccounts.java

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2024 Google LLC
```

Comment:
Please revert the copyright year for all existing snippets. Only brand new ones should have 2024.